modules/signing_worker/manifests/init.pp

```diff
@@ -71,16 +73,20 @@
 
     $required_directories = [
       $scriptworker_base,
-      "${scriptworker_base}/certs",
       "${scriptworker_base}/logs",
-      "${scriptworker_base}/artifact",
```

Comment:
`artifact` is unused, and scriptworker creates the `artifacts` directory on start.